### ISSUE TITLE
Logs file not found

### ISSUE THREAD

**Issue by efraimip** [lemasato/POE-Trades-Companion#214]
I am running POE on a ramdisk using ImDisk.
POE Trade Companion doesnt seem to work with POE on a ramdisk.
When loaded, it gives the error "Logs file not found" and says "\Device\ImDisk0\Path of Exile\logs\Client.txt not found"

Rest assured that the client.txt file is alive and well in there. Other programs that read from it work just fine.

I use a symbolic link (mklink /D) to point D:\games\Path of Exile --> R:\Path of Exile
Everything else works, have never seen this type of error before, but seems that POETC somehow finds out that it is a symlink, which it isnt supposed to. As far as file system goes, symlink gives a file header that is unidentifiable as a link (every program should think it is the hard path).

Any fix for this would be hugely appreciated as POE runs 1000x better on RAMDISK, but I cant imagine playing POE without the POETC.

Thanks!

**Comment by lemasato** [comment]
Just tried and seems to work fine. The tool detects the game location without any issue.  

I used [this RAMDISK](https://www.softperfect.com/products/ramdisk/).  
To make testings easier (and not having to create a 15gb RAMDISK), I'm using a small dummy exe along with a /logs/Client.txt file. Though, that shouldn't matter as the command used retrieve the game path directly from the process.  
Tried both normally and with a symbolic link.  

I recommend to use the /GameFolder [cmd-line parameter](https://github.com/lemasato/POE-Trades-Companion/wiki/Command-Line-Parameters) and point it to your game folder. The tool will use that location instead of automatically retrieving it from the process.

**Comment by efraimip** [comment]
Hi lemasato

Tried your suggestion but it doesnt work. Tried running from the command line and creating a shortcut with the /GamePath "R:\Path of Exile" but no luck.
It still gives the same error.

I cant use The RAMDISK program you tried because it requires a license to purchase after 30 days (I used to use that program for yeaaaars, then they started charging for it sadly).
Could be an issue with ImDisk, or could be a system issue, not sure, but POETC doesnt work in any case and thats a game breaker for me! :|

If you dont have any other ideas I'll try figuring it out.
